### PR TITLE
Add Longitude gtest unit test

### PR DESCRIPTION
**Pull request by chrisryancombs**

### CODE REVIEW

File: isis/src/base/objs/Longitude/LongitudeTests.cpp [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+  EXPECT_FALSE(lon.inRange(Longitude(360, Angle::Degrees), Longitude(0, Angle::Degrees)));
+  EXPECT_FALSE(lon.inRange(Longitude(350, Angle::Degrees), Longitude(355, Angle::Degrees)));
+  EXPECT_TRUE(lon.inRange(Longitude(0, Angle::Degrees), Longitude(160, Angle::Degrees)));
+  EXPECT_FALSE(lon.inRange(Longitude(0, Angle::Degrees), Longitude(44, Angle::Degrees)));
+  EXPECT_FALSE(lon.inRange(Longitude(46, Angle::Degrees), Longitude(90, Angle::Degrees)));
+  EXPECT_TRUE(lon.inRange(Longitude(0, Angle::Degrees), Longitude(45, Angle::Degrees)));
+  EXPECT_TRUE(lon.inRange(Longitude(45, Angle::Degrees), Longitude(90, Angle::Degrees)));
+}
+
+
```
Comment (krlberry) [inline isis/src/base/objs/Longitude/LongitudeTests.cpp]: Very minor comment, but could you remove all of the extra lines at the end of this file.

File: isis/src/base/objs/Longitude/LongitudeTests.cpp [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+// Constructor given a value in degrees
+TEST(Longitude, DegreesConstructor) {
+  Longitude lon(180.0, Angle::Degrees);
+  EXPECT_EQ(180, lon.degrees());    
+}
+
+// Constructor given a positive west value in degrees
+TEST(Longitude, PositiveWest) {
+  Longitude lon(180.0, Angle::Degrees, Longitude::PositiveWest);
```
Comment (kaitlyndlee) [inline isis/src/base/objs/Longitude/LongitudeTests.cpp:24]: Should we also check to see what the methods positiveWest() and positiveEast() return?

File: isis/src/base/objs/Longitude/LongitudeTests.cpp [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+  EXPECT_EQ(450, lon.degrees());
+}
+
+// Copy constructor
+TEST(Longitude, CopyConstructor) {
+  Longitude lon(-90.0, Angle::Degrees, Longitude::PositiveWest,
+        Longitude::Domain360);
+  EXPECT_EQ(lon.degrees(), Longitude(lon).degrees());
+}
+
```
Comment (kaitlyndlee) [inline isis/src/base/objs/Longitude/LongitudeTests.cpp:51]: Do we need to test the other two constructors?

File: isis/src/base/objs/Longitude/LongitudeTests.cpp [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+// ----- Testing Set Methods -----
+// Set to 90 degrees
+TEST(Longitude, Set90Degrees) {
+  Longitude lon(270.0, Angle::Degrees);
+  lon.setPositiveEast(90, Angle::Degrees);
+  EXPECT_EQ(90, lon.degrees());
```
Comment (kaitlyndlee) [inline isis/src/base/objs/Longitude/LongitudeTests.cpp:57]: Should we also check to see what positiveEast() returns?

File: isis/src/base/objs/Longitude/LongitudeTests.cpp [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+}
+
+// Set to 90 degrees PW
+TEST(Longitude, Set90DegreesPW) {
+  Longitude lon(270.0, Angle::Degrees);
+  Longitude lonCopy(lon);
+  lon.setPositiveWest(90, Angle::Degrees);
+  EXPECT_EQ(270, lon.degrees());
```
Comment (kaitlyndlee) [inline isis/src/base/objs/Longitude/LongitudeTests.cpp:64]: Should we also check to see what positiveWest() returns?

File: isis/src/base/objs/Longitude/LongitudeTests.cpp [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+
+  lonCopy = lon;
```
Comment (kaitlyndlee) [inline isis/src/base/objs/Longitude/LongitudeTests.cpp]: Should this be in its own test since we are testing an operator?

**Comment by krlberry** [comment]
@chrisryancombs Could you put the old unit test and truthdata back in? We're leaving them in for now.

**Review by krlberry — CHANGES_REQUESTED** [review]
@chrisryancombs Could you put the old unit test and truthdata back in? We're leaving them in for now.

**Comment by Kelvinrr** [comment]
Can one of the admins verify this patch?

**Comment by jlaura** [comment]
@chrisryancombs Can you update this for the comments from @kberryUSGS and then change the target branch to `dev`?

**Comment by jessemapel** [comment]
As Ryan has left, continued at #3182

**Pull request closed without merging**